d2graph/d2graph.go

```diff
@@ -331,100 +331,100 @@ func (l ContainerLevel) LabelSize() int {
 	return d2fonts.FONT_SIZE_M
 }
 
-func (obj *Object) GetFill(theme *d2themes.Theme) string {
+func (obj *Object) GetFill() string {
```

Comment:
+1 nice, i do think it makes sense to separate out the color coding and the actual colors.